### PR TITLE
Added example for ``path_arc`` keyword argument of :class:`.Transform`

### PR DESCRIPTION
**Pull request by kjlubick**
## Overview: What does this pull request change?
This was very handy for me when trying to swap two parts of an equation around without them getting muddled together first. Thus, I want it to be more clear how to use the `path_arc` feature of Transform, so I added an example showing how the different angles change the movement path.

## Links to added or changed documentation pages
[Transform Page](https://manimce--2849.org.readthedocs.build/en/2849/reference/manim.animation.transform.Transform.html#manim.animation.transform.Transform)

https://user-images.githubusercontent.com/6819944/175700022-efb9c6b3-d278-4905-8bc1-85c8c093c19a.mp4

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Comment by MrDiver** [comment]
This looks good. Thank you for your work! But i'm not sure if we can support more Animations in the Documentation right now. @behackl might take a look in a few days. Till then we will just keep this here till we know for sure.

**Comment by Kiran-Raj-Dev** [comment]
But you can add it as a template though!
https://github.com/ManimCommunity/manim/tree/main/manim/templates

File: manim/animation/transform.py [hunk 1]
```diff
@@ -74,6 +75,49 @@ class Transform(Animation):
...
+                        points.append(p_fn(start, end, alpha / 100.0))
+                    path = VMobject(stroke_color=YELLOW)
+                    path.set_points_smoothly(points)
+                    return path
+
+                left = Circle(stroke_color=BLUE_E, fill_opacity=1.0, radius=0.5).move_to(LEFT * 2)
+                colors = [TEAL_A, TEAL_B, TEAL_C, TEAL_D, TEAL_E, GREEN_A]
+                # Positive angles move clockwise, negative angles move counter-clockwise.
+                examples = [-90, 0, 30, 90, 180, 270]
+                arcs, circles, text, anims = [], [], [], []
```
[the remark this answers is not in the document]
Comment (kjlubick) [reply manim/animation/transform.py:101]: The reason I did it this way was to make sure the circles were on top, the text was behind them, and the arcs were behind them. I tried that a different way using `set_z_index`, what do you think?

File: manim/animation/transform.py [hunk 2]
```diff
@@ -74,6 +75,49 @@ class Transform(Animation):
...
+                for idx in range(0, len(examples)):
```
[the remark this answers is not in the document]
Comment (kjlubick) [reply manim/animation/transform.py:102]: Done.

File: manim/animation/transform.py [hunk 3]
```diff
@@ -74,6 +75,49 @@ class Transform(Animation):
...
+    .. manim :: TransformPathArc
+
+        class TransformPathArc(Scene):
+            def construct(self):
+                def make_arc_path(start, end, arc_angle):
+                    points = []
+                    p_fn = path_along_arc(arc_angle)
+                    # alpha animates between 0.0 and 1.0, where 0.0
+                    # is the beginning of the animation and 1.0 is the end.
+                    for alpha in range(0, 100):
```
[the remark this answers is not in the document]
Comment (kjlubick) [reply manim/animation/transform.py:91]: Done.

**Comment by kjlubick** [comment]
Thank you for the excellent suggestions.

**Comment by kjlubick** [comment]
Anything else to address?

**Review by behackl — APPROVED** [review]
LGTM, thank you!

**Pull request merged**